pvt71/components/__tests__/Timer-test.js

```diff
@@ -0,0 +1,68 @@
+import 'react-native';
+import React from 'react';
+import { Timer } from '../Timer';
+import renderer from 'react-test-renderer'
+
+describe('Timer Test', () => {
+    describe('Timer Snapshot', () =>{
```

Comment:
Use `it` instead of `describe`. then, `it('should render to snapshot', () => { })` it makes naming much easier.